orElse callback when retries are exhausted (#167)
google/dart-neats: retry/CHANGELOG.md

## v3.2.0

>  * `orElse` callback to calculate a value when the attempts where exhausted.

Reviewer: ```suggestion
 * Introduced `orElse` callback to create a value when the retry attempts are exhausted.
```

Author: Hm, yeah, it may be confusing. Maybe `onRetryExhausted` could be used instead?
